src/main/java/ChooseAccountPage.java

```diff
@@ -0,0 +1,40 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class ChooseAccountPage {
+
+    private final WebDriver driver;
+    private final By youAreLoggedOutText = By.xpath("//h3[@class='AuthAccountList-unathorizedAccountsTitle']");
+    private final By accountsList = By.xpath("//div/a[@class='AuthAccountListItem']");
+    private final By currentAccountName = By.xpath("//span[@class='CurrentAccount-displayName']");
+
+    public ChooseAccountPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public String getLogoutText() {
+        WebElement account = driver.findElement(currentAccountName);
+        account.click();
+        WebDriverWait wait = new WebDriverWait(driver, 10);
+        wait.until(ExpectedConditions.visibilityOfElementLocated(youAreLoggedOutText));
+        return driver.findElement(youAreLoggedOutText).getText();
+    }
+
+    public List<WebElement> getAccountsList() {
```

Comment:
If public method in page object class returns WebElement - this signals that here you have error. Sense of page object - encapsulate elements and business functions. If you return WebElement, then you provide direct access to element on the page

Reply:
Actually, this method is not needed so I decided to remove it